index.php

```diff
@@ -1,9 +1,10 @@
+<h1>Welcome to my collection of records</h1>
```

Comment:
Could this HTML go beneath the PHP section? As you expand the HTML and start referencing variables from PHP, you may want the PHP to run before the HTML